Resources/migration.yml

```diff
@@ -335,3 +335,8 @@ ReinforcementRadioSyndicateMonkeyNukeops: ReinforcementRadioSyndicateAncestorNuk
 
 # 2024-05-01
 DrinkBottleGoldschlager: DrinkBottleGildlager
+
+# 2024-05-11
+FloorTileItemGrassJungle: FloorTileItemGrass
+FloorTileItemMowedAstroGrass: FloorTileItemAstroGrass
+FloorTileItemJungleAstroGrass: FloorTileItemAstroGrass
```

Comment:
You need to use tile alises for tiles, not migrations.

Reply:
these are the floor tile items, not the tiles themselves